components/badge/badge.css

```diff
@@ -1,42 +1,38 @@
-
-
 .badge {
-    width: min-content;
-    height: min-content;
-    position: relative;
+  width: min-content;
+  height: min-content;
+  position: relative;
 }
 
-
 .badge-count {
-    font-size: 0.5em;
-    padding: 0.1rem 0.35rem;
-    border-radius: 50%;
-    background-color: rgb(0, 255, 76);
-
-    position: absolute;
-    top: 0%;
-    right: 0%;
-    transform: translate(50%, -50%);   
+  font-size: 1.2em;
+  padding: 0.2rem 0.5rem;
+  border-radius: 50%;
+  background-color: var(--info__bg-color);
+
+  position: absolute;
+  top: 0%;
+  right: 0%;
+  transform: translate(50%, -50%);
 }
 
-
 .container-badges {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    gap: 10px;
-    height: 100vh;
-    width: 100vw;
-
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  height: 100vh;
+  width: 100vw;
 }
 
 .badge-text {
-    font-size: 1em;
-    padding: 0.1rem 0.35rem;
-    border-radius: 0.05em;
-    position: absolute; 
-    top: 0%;
-    right: 0%;
-    transform: translate(50%, -50%); 
+  font-size: 1em;
+  padding: 0.1rem 0.35rem;
```

Comment:
Is there any reason to give padding in rems here, we can give it in px right if it is less than 10px